python_src/poetry.lock

```diff
@@ -0,0 +1,1008 @@
+[[package]]
```

Comment:
classic poetry lock file for the new package. it stays checked in to make sure everything is the same version.